x/emissions/types/events.go

```diff
@@ -22,6 +24,7 @@ func EmitNewForecasterScoresSetEvent(ctx sdk.Context, scores []Score) {
 	if len(scores) < 1 {
 		return
 	}
+	metrics.IncrProducerEventCount(metrics.FORECASTER_SCORE_EVENT)
 	err := ctx.EventManager().EmitTypedEvent(NewScoresSetEventBase(ActorType_ACTOR_TYPE_FORECASTER, scores))
```

Comment:
I don't think we need to keep both... cc @kpeluso

Reply:
I'm not opposed to what @RedBird96 is doing here actually -- this provides a decent sanity check for any indexer nop